src/directories.cc

```diff
@@ -540,7 +540,7 @@ void Directories::add_or_update_path(const boost::filesystem::path &dir_path, co
   if(path_it==directories.end()) {
     auto g_file=Glib::wrap(g_file_new_for_path(dir_path.string().c_str())); //TODO: report missing constructor in giomm
 
-#if defined(__FreeBSD__)
+#if defined(__OpenBSD__) || defined(__NetBSD__) || defined(__FreeBSD__) || defined(__Darwin__) || defined(__DragonFly__)
```

Comment:
Thank you, but do you know if this check is needed for other Unix variants than FreeBSD? I though it was an exception for FreeBSD, but I might be mistaken.